src/inline.d

```diff
@@ -35,8 +35,10 @@ import ddmd.tokens;
 import ddmd.visitor;
 
 private:
-enum LOG = false;
-enum CANINLINE_LOG = false;
+
+enum LOG = 0;
+enum CANINLINE_LOG = 0;
+enum EXPANDINLINE_LOG = 0;
```

Comment:
However, I can personally use `0`/`1` during debugging, then can revert to `false` after my work. So, change back to use `false` for code readability.